src/lib/flexbox/api/base.ts

```diff
@@ -92,6 +91,15 @@ export abstract class BaseFxDirective implements OnDestroy, OnChanges {
   // Lifecycle Methods
   // *********************************************
 
+  /**
+   * Use post-component-initialization event to perform extra
+   * querying such as computed Display style
+   */
+  ngOnInit() {
+    this._display = this._getDisplayStyle();
+    this._hasInitialized = true;
```

Comment:
`hasInitialized` is only used in a `ngDoCheck` hook. [According to this chart](https://angular.io/guide/lifecycle-hooks), it shouldn't be possible for `ngDoCheck` to fire before `ngOnInit`, making the flag redundant.